setup.py

```diff
@@ -25,6 +25,8 @@
     install_requires=[
         "django>=1.8",
         "django-model-utils==1.4.0",
+        "djangorestframework==3.2.3",
```

Comment:
These should not be hard requirements. Packages need to be able to work with multiple versions: `djangorestframework>=3.2.0,<3.4.0` (I *think* 3.3.x should work.)